tests/FakeItEasy.IntegrationTests/FakeItEasy.IntegrationTests.csproj

```diff
@@ -39,7 +39,7 @@
     <ProjectReference Include="..\..\src\FakeItEasy\FakeItEasy.csproj" />
     <ProjectReference Include="..\..\tests\FakeItEasy.Tests\FakeItEasy.Tests.csproj" />
     <ProjectReference Include="..\FakeItEasy.IntegrationTests.External\FakeItEasy.IntegrationTests.External.csproj">
-      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+      <Private>false</Private>

```

Comment:
Damn! It doesn't work for .NET Core, because the deps.json file knows about the External assembly, and fails when it tries to load it (we should probably catch that error, btw... it happens [here](https://github.com/FakeItEasy/FakeItEasy/blob/5880552eeca974d9c4ed072163e80eb5e052415d/src/FakeItEasy/Core/TypeCatalogue.cs#L82))...
I guess I'll just remove the dependency completely. The only downside is that it will be possible to run the integration tests in VS without building the External assembly first.